Rubberduck.Parsing/ComReflection/ComParameter.cs

```diff
@@ -37,7 +38,7 @@ public ComParameter(ITypeInfo info, ELEMDESC elemDesc, string name, int index) :
             var paramDesc = elemDesc.desc.paramdesc;
             IsOptional = paramDesc.wParamFlags.HasFlag(PARAMFLAG.PARAMFLAG_FOPT);
             IsReturnValue = paramDesc.wParamFlags.HasFlag(PARAMFLAG.PARAMFLAG_FRETVAL);
-            IsByRef = (VarEnum) elemDesc.tdesc.vt == VarEnum.VT_PTR;
+            IsByRef = Convert.ToBoolean(elemDesc.tdesc.vt & (int)VarEnum.VT_PTR);
```

Comment:
I believe this test should be more like:   vt & VT_TYPEMASK == VT_PTR

Reply:
Fixed.